syntax = "proto3";
package protos.playbooks;

import "protos/base.proto";
import "google/protobuf/wrappers.proto";


message PlaybookBashCommandTask {
  google.protobuf.StringValue command = 1;
  google.protobuf.StringValue remote_server = 2;
}